Source/JavaScriptCore/jit/GCAwareJITStubRoutine.cpp

```diff
@@ -163,6 +163,15 @@ void PolymorphicAccessJITStubRoutine::addedToSharedJITStubSet()
     m_isInSharedJITStubSet = true;
 }
 
+bool PolymorphicAccessJITStubRoutine::visitWeakImpl(VM& vm)
+{
+    bool isValid = true;
+    for (StructureID weakReference : m_weakStructures)
+        isValid &= vm.heap.isMarked(weakReference.decode());
+    isValid &= GCAwareJITStubRoutine::visitWeakImpl(vm);
```

Comment:
Nit: can we do `using Base = GCAwareJITStubRoutine;` for this instead of naming the base class directly.